### PR TITLE
First try of update

### PR DESCRIPTION
**Pull request by yvanlebras**
Is this manner to start acceptable ?

### CODE REVIEW

File: NGS-QC/tutorials/dive_into_qc.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ General introduction about the topic and then an introduction of the tutorial (t
 
 # Check the quality of your sequences
 
-Introduction about this part
+Chacking sequences quality, and raw data quality in general, is one of the most important part of an analysis. Taking time to evaluate deeply each part of your data can give a lot of benefit on the next steps and avoids loosing time re-executing several times the entire workflow.
```
Comment (hrhotz) [inline NGS-QC/tutorials/dive_into_qc.md]: typo: Checking

File: NGS-QC/tutorials/dive_into_qc.md [hunk 2]
```diff
@@ -52,21 +52,29 @@ Short introduction about this subpart.
 
 ## Check the FastQC output
 
-Short introduction about this subpart.
+FastQC outputs are an html report and a raw data file. If the raw data file allows the user to start from original data to create its own grpahs and stats and the html report shows directly a graphical web page, the two outputs are divided in sections :
 
-:pencil2: ***Hands on!***
+:pencil2: ***Basics statistics***
```
Comment (bebatut) [inline NGS-QC/tutorials/dive_into_qc.md]: The "Hands on" must stay and after we would like tasks to do. If you want a subsection, use `### Basics statistics`

File: NGS-QC/tutorials/dive_into_qc.md [hunk 3]
```diff
@@ -52,21 +52,29 @@ Short introduction about this subpart.
...
+-the ASCII encoding type, relevant when using FastQ groomer next to FastQC to convert your fastQ files to the fastqsanger format.
+-number of sequences
+-sequence length
+-the overall GC content
 
-Some blabla
+:pencil2: ***Per base sequence quality ***
 
-:pencil2: ***Hands on!***
+A red line indicates the mean value of the quality score when the yellow box allows the user to determine inter-quartiles values (25% - 75%). box plot top and bottom show respectively the 90% and 10% treshold. The 3 parts of the graph (green, orange, red), represent respectively very good, quite good and not so good ;) quality. It's classical to see the overall score decreasing with le sequence length as sequencing qulity is decreasing with time.
```
Comment (bebatut) [inline NGS-QC/tutorials/dive_into_qc.md]: It could be nice also to add an illustration (a FastQC output)

**Comment by bgruening** [comment]
@yvanlebras can we close this or is still valid :)

**Pull request closed without merging**